content/applications/hr/referrals/rewards.rst

```diff
@@ -0,0 +1,76 @@
+=======
+Rewards
+=======
+
+After employees have successfully earned referral points, they can exchange their points by
+purchasing rewards in Odoo's *Referrals* application. Rewards **must** be :ref:`created and
+configured <referrals/create>` before employees can :ref:`redeem points for rewards
+<referrals/redeem>`.
+
+.. _referrals/create:
+
+Create rewards
+==============
+
+Rewards are the only configurations needed when setting up the *Referrals* application. Only users
+with :guilabel:`Administrator` rights for the *Recruitment* application can create or modify
+rewards. To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`.
+Click :guilabel:`New`, and a reward form loads. Enter the following information on the form:
+
+- :guilabel:`Product Name`: enter the name as it should appear for the reward.
+- :guilabel:`Cost`: enter the amount of points required to redeem the reward.
+- :guilabel:`Company`: using the drop-down menu, select the company the reward appears for. If a
+  reward should be used for several companies, each company needs to have a record of the reward,
+  with the company listed on the form. This field **only** appears if in a multi-company
+  environment.
+- :guilabel:`Gift Responsible`: select the person responsible for procuring and delivering the
+  reward to the recipient, using the drop-down menu. This person is alerted when the reward is
+  bought in the *Referrals* application, so they know when to deliver the reward to the recipient.
+- :guilabel:`Description`: type in the description for the reward. This is visible on the reward
+  card, beneath the title.
+- :guilabel:`Photo`: add a photo of the reward, which appears on the rewards page. Hover over the
+  image square, and a :icon:`fa-pencil` :guilabel:`(pencil)` icon appears. Click the
+  :icon:`fa-pencil` :guilabel:`(pencil)` icon, and a file explorer appears. Navigate to the photo
+  file, then click :guilabel:`Open` to select it. Once a photo is selected, hovering over the image
+  reveals two icons instead of one: a :icon:`fa-pencil` :guilabel:`(pencil)` icon and a
+  :icon:`fa-trash-o` :guilabel:`(trash can)` icon. Click the :icon:`fa-trash-o` :guilabel:`(trash
+  can)` icon to delete the currently selected image.
```

Comment:
couple things here...

...and these are purely SUGGESTIONS...

- you may want to consider placing this explanation *before* the 'Description' tab, as it appears alongside the other fields at the top of the form
- perhaps, instead of 'image square' some more descriptive language could be used here to visually describe what it looks like (camera icon and plus sign icon, or something along those lines)
- While I fully appreciate the attention to detail here explaining how to upload an image, there may be a more succinct way to explaining it. For instance, I'm sure it's necessary to explain the appearance of two icons after its uploaded, and what they do...we may be able to assume the reader/user would understand what would happen if they clicked the 'trash can' icon, as that is, usually, a pretty universal icon...

Again, these are just SUGGESTIONS to consider during your revision 👍